nilearn/connectome/connectivity_matrices.py

```diff
@@ -284,13 +358,17 @@ class ConnectivityMeasure(BaseEstimator, TransformerMixin):
             "covariance", "precision"}, optional
         The matrix kind.
 
+    vectorize : bool, optional
+        Vectorize or not the connectivities.
+
     Attributes
     ----------
     `cov_estimator_` : estimator object
         A new covariance estimator with the same parameters as cov_estimator.
 
     `mean_` : numpy.ndarray
-        The mean connectivity for the tangent kind.
+        The mean connectivity across subjects. Note that for 'tangent' kind,
+        averaging procedure is not the trivial one.
```

Comment:
So which means you are not returning mean_ for kind='tangent' ? May be you could be explicit here ?